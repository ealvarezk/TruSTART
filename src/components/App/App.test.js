import React from 'react';
import { shallow } from 'enzyme';
import App from '.';

test('renders App properly', () => {
  const app = shallow(<App />);
  expect(app).toMatchSnapshot();
});
