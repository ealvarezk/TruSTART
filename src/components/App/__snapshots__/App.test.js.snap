// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App properly 1`] = `
<div
  className="App"
>
  <Projects />
</div>
`;
